src/lib/theme/_palette.scss

```diff
@@ -149,10 +148,10 @@ $td-blue: (
 
 // Background palette for light themes.
 $mat-light-theme-background: (
-  status-bar: #e0e0e0,
-  app-bar: mat-color($mat-grey, 50),
-  background: mat-color($mat-grey, 200),
-  card: white,
+  status-bar: mat-color($mat-grey, 400),
```

Comment:
Need to revert this

Reply:
But we can use mat-color instead of the hex, so

`status-bar: mat-color($mat-grey, 300),`